exercises/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,37 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
```

Comment:
The same principle applies here with the statements in this folder:
`return num < 0;`